### PR TITLE
Get Programming with Haksell

### PR DESCRIPTION
**Pull request by sjanasek**
added another recommendation "Get Programming with Haksell" by Will Kurt.

- [ x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x ] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x ] My pull request targets the `master` branch of freeCodeCamp.
- [x ] None of my changes are plagiarized from another source without proper attribution.
- [ x] My article does not contain shortened URLs or affiliate links.

If your pull request closes a GitHub issue, replace the XXXXX below with the issue number.

Closes #XXXXX

### CODE REVIEW

File: guide/english/book-recommendations/haskell/index.md [hunk 1]
```diff
@@ -31,3 +31,9 @@ Available to buy or download for free (e-book)! This book is broken down into tw
 
 - [Website](http://haskellbook.com/index.html)
 - [Gumroad](https://gumroad.com/l/haskellbook)
+
+#### Get Programming with Haksell
```
Comment (CodingSolo) [inline guide/english/book-recommendations/haskell/index.md]: Please correct this line `Haskell` not Haksell

File: guide/english/book-recommendations/haskell/index.md [hunk 2]
```diff
@@ -31,3 +31,9 @@ Available to buy or download for free (e-book)! This book is broken down into tw
...
+
+This book introduces you to the Haskell language without drowning you in academic jargon or heavy programming theory. It's great for beginners. It is written by Will Kurt. It is packed with focused topics, short code examples and short programs, plus numerous exercises to help stretch and strengthen your new knowledge.
+
+-[Amazon](https://www.amazon.com/Get-Programming-Haskell-Will-Kurt/dp/1617293768)
```
Comment (CodingSolo) [inline guide/english/book-recommendations/haskell/index.md]: Please and a space between `-` and `[`

**Comment by CodingSolo** [comment]
LGTM.

Thank you for PR.

**Comment by moT01** [comment]
Thank you for the PR @sjanasek, congratulations on your first contribution to freeCodeCamp! We look forward to future contributions!

**Pull request merged**